dbeaver/dbeaver
# dbeaver/dbeaver#22325 add opens sun.nio.fs to handle symlinks

Closes dbeaver/dbeaver#22325

To test:
Create a symlink to the desired workspace in Powershell in Administrator mode
```
New-Item -ItemType SymbolicLink -Path {LinkLocation} -Target {workspaceLocation}
```
When try to launch from the symbolic link and see if it works
